### ISSUE TITLE
[A11Y] Add accessibility support for ImageRegionSelectionInteraction [Blocked: oppia/#9924]

### ISSUE THREAD

**Issue by FareesHussain** [oppia/oppia-android#3712]
ImageRegions of ImageRegionSelectionInteraction doesn't have accessibility support.

Additional information other than the exact answer need to be added as content description for each image region.

https://github.com/oppia/oppia/issues/9924 For details

**Comment by vrajdesai78** [comment]
Blocked on [#9924](https://github.com/oppia/oppia/issues/9924#issuecomment-1077043567)

**Comment by BenHenning** [comment]
Looks like a TODO wasn't removed @vrajdesai78.

**Comment by vrajdesai78** [comment]
@BenHenning will I have to create a new pull request to remove this TODO ?

**Comment by BenHenning** [comment]
In order to close the issue, yes @vrajdesai78. Since it's trivial, you could add it to another ongoing PR to avoid creating a new PR (either approach is fine, and is up to you).

**Comment by vrajdesai78** [comment]
@BenHenning created new PR #4543
